swimlane/core/resources/report.py

```diff
@@ -111,6 +112,18 @@ def filter(self, field_name, operand, value):
             raise ValueError('Operand must be one of {}'.format(', '.join(self._FILTER_OPERANDS)))
 
         field = self._get_stub_field(field_name)
+        
+        accepted_values_to_check = (int, str, float, list, bool, tuple)
```

Comment:
Can we extract this into a function and use it in `report.py` and `record.py`?

Reply:
where can I place this function? I looked for a helpers/utils function that both could share, but I couldn't find anything obvious.

Reply:
adding it to utils/__init__.py